coursefinder/templates/utils.py

```diff
@@ -0,0 +1,149 @@
+from CMS.test.mocks import StatusMocks
+from errors.models import InternalError
+
+
+def choose_country_sibling_finder(pages_list):
+    def is_choose_country(sibling):
+        from coursefinder.models import CourseFinderChooseCountry
+
+        if type(sibling) == CourseFinderChooseCountry:
+            return True
+        else:
+            return False
+
+    choose_country_page = list(filter(is_choose_country, pages_list))
+
+    if len(choose_country_page) == 0:
```

Comment:
I don't think there is any need for len here and elsewhere as an empty list returns False anyway.